src/Lecture1.hs

```diff
@@ -63,7 +65,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integer -> Integer
+lastDigit n = if n >=0 then mod n 10 else mod (-n) 10
```

Comment:
Good catch on the negative case!
You can also handle both cases at once with the usage of the [`abs`](https://hackage.haskell.org/package/base-4.16.0.0/docs/Prelude.html#v:abs) function 🔤